.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea/
```

Comment:
Probably. Anyway, I don't think it's worth a commit. It can be removed in the next commit if it shouldn't be there.